### ISSUE TITLE
[FEED] move hosts only on creation

### ISSUE THREAD

**Issue by FloMiau** [tribe29/ansible-collection-tribe29.checkmk#102]
Please verify first that your feedback is not already reported [here](https://github.com/tribe29/ansible-collection-tribe29.checkmk/issues?q=is%3Aissue+sort%3Aupdated-desc).
If possible and useful provide additional information below. This is optional however.

**Provide your feedback**

Hi there,

I'm using the collection to add hosts to my monitoring. I've got an issue with hosts that are already monitored.
Since the folder attribute isn't optional and has an default value of / the code on https://github.com/tribe29/ansible-collection-tribe29.checkmk/blob/devel/plugins/modules/host.py#L287-L289 will be executed:

```
        if current_folder != module.params["folder"]:
            move_host(module, base_url, headers)
            msg_tokens.append("Host was moved.")
```
In my environment I have to choose the folder manually so the module will move the host into the wrong folder.

Is it possible to place the host into a folder only when the host is created?

Additionally there should be an option to retrieve the folder via this module. For example with a "state: info" so I can query the folder before.

**Component Name**

`host`

**Ansible Version**

```console
$ ansible --version
ansible [core 2.12.7]

```

**Comment by robin-tribe29** [comment]
Hi @FloMiau, only placing a host on creation into a folder would violate the whole purpose of Ansible and idempotency. So we will not do that. Your second query would be a lookup plugin I think. But your suggestion of implementing a `state: info` or similar is not the Ansible way. Additionally, you would define the desired state in your inventory, rather than looking up, what is going on in Checkmk.

This is my point of view however, I will gladly discuss your take on this.

**Comment by robin-tribe29** [comment]
The underlying issue was fixed in #147. As there was no response to this day, I will close this issue as solved.
